TFNetwork.py

```diff
@@ -2071,6 +2071,10 @@ def _tf_summary(self):
         tf.summary.scalar("loss_%s" % name, self._loss_value * self._norm_factor)
         if self._network.get_config().bool("calculate_exp_loss", False):
           tf.summary.scalar("exp_loss_%s" % name, tf.exp(self._loss_value * self._norm_factor))
+        if self._network.get_config().bool("debug_unnormalized_loss_summaries", False):
+          tf.summary.scalar("unnormalized_loss_%s" % name, self._loss_value)
+        if self._network.get_config().bool("debug_objective_loss_summaries", False):
+          tf.summary.scalar("objective_loss_%s" % name, self._loss_value_for_objective)
```

Comment:
Did you test this? Do you actually use this? I think this does not work, as `_loss_value_for_objective` is not set at this point. I would just remove it, if you don't use it. Don't just add random features.
See where `_tf_summary` is called. The logic currently makes sure that `_loss_value` at that point is actually exactly the loss used for the objective. I think this option is confusing.

Reply:
Ah, I'm just seeing, you moved the `_tf_summary` call.
But anyway, I'm not really sure why you would need `objective_loss_...`. How is this useful? You either have the information in `loss_...` or `unnormalized_loss_...`.

Reply:
Normalized and unnormalized do not include the loss scaling. I use pretraining to modify the loss scaling over time, and i want to now the actual relation of the losses during the training. And for me it seemed only `_loss_value_for_objective` contains the exact loss for gradient computation in all cases.

Reply:
Ah I see, ok.